### PR TITLE
bpo-33138: Change standard error message for non-pickleable and non-copyable types.

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
It now says "cannot serialize" instead of "can't pickle".

https://bugs.python.org/issue33138

### CODE REVIEW

**Comment by serhiy-storchaka** [comment]
I'm going to merge this PR if there are no other suggestions.

File: Lib/copyreg.py [hunk 1]
```diff
@@ -53,24 +53,27 @@ def _reconstructor(cls, base, state):
...
+    args = (cls, base, state)
     try:
         getstate = self.__getstate__
     except AttributeError:
         if getattr(self, "__slots__", None):
-            raise TypeError("a class that defines __slots__ without "
-                            "defining __getstate__ cannot be pickled") from None
+            raise TypeError(f"cannot serialize '{cls.__name__}' object: "
+                            f"a class that defines __slots__ without "
+                            f"defining __getstate__ cannot be pickled"
```
Comment (ZackerySpytz) [inline Lib/copyreg.py]: There is a missing space here.

Comment (serhiy-storchaka) [reply Lib/copyreg.py]: Sorry, could you be more specific?

Comment (serhiy-storchaka) [reply Lib/copyreg.py]: Ah, I see, the ending space on this line. Thanks!

**Review by methane — APPROVED** [review]

**Review by vstinner — APPROVED** [review]
LGTM.

**Pull request merged**